src/model/Result.js

```diff
@@ -0,0 +1,109 @@
+const { GAME } = require('../constant/Constants');
+
+class Result {
+  #rank;
+
+  #count;
+
+  #bonusCorrect;
+
+  constructor() {
+    this.#rank = [0, 0, 0, 0, 0];
```

Comment:
해당 배열의 각 인덱스가 각 로또 당첨 등수를 나타내는 것 같은데요~
코드를 처음 보는 사람이 rank배열의 의미를 파악하려면,
아래 계산 로직이 담겨있는 코드를 읽어봐야 이해할 수 있을 것 같아요.
 
 ```suggestion
    this.#rank ={
       firstPrize: 0,
       secondPrize: 0
       ...
    }
```
위와 같은 객체 형태나 Map 자료구조는 어떨까요?